pennylane/ops/functions/bind_new_parameters.py

```diff
@@ -111,6 +111,20 @@ def bind_new_parameters_composite_op(op: CompositeOp, params: Sequence[TensorLik
     return op.__class__(*new_operands)
 
 
+@bind_new_parameters.register
+def bind_new_parameters_cy(
```

Comment:
Might be worth just updating the main `bind_new_parameters` to return a copy if the number of parameters is 0